bluemira/equilibria/run.py

```diff
@@ -196,35 +355,35 @@ def run_reference_equilibrium(self):
             coilset,
             eq,
             MagneticConstraintSet(self.eq_constraints),
-            gamma=self._eq_settings["gamma"],
+            gamma=self._eq_settings.gamma,
         )
         program = PicardIterator(
             eq,
             opt_problem,
             convergence=self._eq_convergence,
-            relaxation=self._eq_settings["relaxation"],
+            relaxation=self._eq_settings.relaxation,
             fixed_coils=True,
             plot=False,
         )
         program()
 
-        eq_constraints = deepcopy(self.eq_constraints)
-        for con in eq_constraints:
-            if isinstance(con, (PsiConstraint, PsiBoundaryConstraint)):
-                eq_constraints.remove(con)
         opt_problem = self._make_opt_problem(
             eq,
             deepcopy(self._eq_opt),
             self._get_max_currents(eq.coilset),
             current_constraints=None,
-            eq_constraints=eq_constraints,
+            eq_constraints=[
+                con
+                for con in deepcopy(self.eq_constraints)
+                if not isinstance(con, (PsiConstraint, PsiBoundaryConstraint))
+            ],
```

Comment:
Very minor, but can't we just copy what we need, rather than the whole list?

```suggestion
            eq_constraints=[
                deepcopy(con)
                for con in self.eq_constraints
                if not isinstance(con, (PsiConstraint, PsiBoundaryConstraint))
            ],
```